### PR TITLE
Update +page.svelte

### PR DESCRIPTION
**Pull request by BBI-Dev**
First line of defence for the prevention of a bug that could be exploited for cheating. Removing this code on client side is possible though.

### CODE REVIEW

**Comment by imdevarsh** [comment]
:eyes:

File: src/routes/(mathex)/mathex/app/play/[id]/+page.svelte [hunk 1]
```diff
@@ -150,6 +150,8 @@
             return;
           }
           socket.emit("answer", answer);
+          document.getElementById(this.id).style.setAttribute("disabled", "");
+          setTimeout( { document.getElementById(this.id).style.removeAttribute("disabled"); } , 16000);
```
Comment (imdevarsh) [inline src/routes/(mathex)/mathex/app/play/[id]/+page.svelte:154]: image
This code has a syntax error, it must be `() => {`
Also it is setting off TypeScript errors, I think i will try and resolve this in a separate way

**Comment by imdevarsh** [comment]
You'll even get a warning if he tries that again :P

**Comment by imdevarsh** [comment]
@BBI-Dev How do you think I should fix the editing situation? Easiest way would be a plain text editor with HTML support (not WYSIWYG like the current one)

**Comment by BBI-Dev** [comment]
I don't know, but maybe something like this: https://experimentbbi.glitch.me

> @BBI-Dev How do you think I should fix the editing situation? Easiest way would be a plain text editor with HTML support (not WYSIWYG like the current one)

**Comment by imdevarsh** [comment]
I decided to just make a textarea for now.

**Pull request closed without merging**